content/languages/cpp/igloo/stringizers.md

```diff
@@ -69,11 +69,11 @@ Test output:
 
 Not only your custom types can be affected by this issue. Snowhouse may be not able to stringify many built-in, standard, or 3rd party types. Basically, every type which does not define its version of the output stream operator (`operator <<`) is affected, and the `std::pair` template is a very common case of such types for Codewars kata.
 
-## Solutions
+### Solutions
```

Comment:
I think it's fine to keep it at `h2`. The sidebar looks kind weird now.

![image](https://user-images.githubusercontent.com/639336/194673650-a77dd16f-4344-40b9-891a-e78df192241b.png)

Current:

![image](https://user-images.githubusercontent.com/639336/194673679-70137b40-e0a7-451e-ba39-734245ba3cfa.png)

Reply:
Maybe `Solutions` header is unnecessary?

Reply:
Good idea. I noticed the sidebar looking worse, but did not think of removing the "Solutions" header. Now I removed it and I think it looks better.